core/src/main/kotlin/io/specmatic/core/filters/ScenarioFilterTags.kt

```diff
@@ -0,0 +1,15 @@
+package io.specmatic.core.filters
+
+import io.specmatic.core.HEADERS_BREADCRUMB
+import io.specmatic.core.METHOD_BREAD_CRUMB
+import io.specmatic.core.PATH_BREAD_CRUMB
+import io.specmatic.core.QUERY_PARAMS_BREADCRUMB
+
+enum class ScenarioFilterTags(val key: String) {
+    METHOD(METHOD_BREAD_CRUMB),
+    PATH(PATH_BREAD_CRUMB),
+    STATUS_CODE("STATUS-CODE"),
```

Comment:
Can you confirm that we do not already have a word for this, by checking in the status matching code HttpResponsePattern.

Reply:
The breadCrumb just says STATUS, shall I change it to STATUS-CODE in breadcrumb in HttpResponsePattern and then use it here?

Reply:
Let this be STATUS for now, but we should change it to STATUS-CODE later.